app/scripts/lib/profile-client.js

```diff
@@ -67,6 +67,16 @@ function (xhr, ProfileErrors) {
     });
   };
 
+  ProfileClient.prototype.getDisplayName = function (accessToken) {
+    return this._request('/v1/display_name', 'get', accessToken);
+  };
+
+  ProfileClient.prototype.postDisplayName = function (accessToken, displayName) {
```

Comment:
I see the same naming convention is used elsewhere, `get`, `post`, `delete`. `post` seems like the odd one out, and is a bit too close for comfort to the HTTP verb for me. `post` doesn't really convey the meaning of what the call does all that well. Is this an update, a create, both?